src/java.base/share/classes/com/sun/crypto/provider/GaloisCounterMode.java

```diff
@@ -1392,17 +1469,11 @@ public int doFinal(byte[] in, int inOfs, int inLen, byte[] out,
             }
 
             if (mismatch != 0) {
+                // Clear output data
+                Arrays.fill(out, outOfs, outOfs + len, (byte) 0);
```

Comment:
Update the javadoc of this method (line 1428-1429) with this change, i.e. clears output data if tag verification fails. Same goes for line 1482-1483.

Reply:
ok